mrjob/aws.py

```diff
@@ -181,6 +181,23 @@
 _DEFAULT_AWS_REGION = 'us-east-1'
 
 
+
+# if AWS throttles us, how long to wait (in seconds) before trying again?
+_AWS_BACKOFF = 20
```

Comment:
This seems too long. Make it 2 s?

Reply:
That is what this revision adds https://github.com/Affirm/mrjob/pull/4